uber/configspec.ini

```diff
@@ -58,6 +58,13 @@ badge_promo_codes_enabled = boolean(default=False)
 # be omitted entirely during preregistration.
 prereg_request_hotel_info_duration = integer(default=0)
 
+# The number of hours after the prereg_request_hotel_info_duration window
+# elapses to wait before the hotel booking link email becomes available to
+# send. Set this to 0 to allow the hotel booking link email to be sent as soon
+# as the prereg_request_hotel_info_duration window elapses.
+# Defaults to 197 hours (8 days and 5 hours).
+prereg_hotel_info_email_wait_duration = integer(default=197)
```

Comment:
Yes and no. Usually this number will be much smaller, like 0 or 24 or something. I would like to start using more relative dates. Maybe use a smart date library that understands things like "two weeks" or "next friday".

Having a default of 197 is kinda crazy, but for the time being, I am fine with this.